script/munin-node-configure

```diff
@@ -328,7 +328,14 @@ sub init_snmp
 sub run_plugin
 {
     my ($plugins, $plugin, $mode) = @_;
-    my $name = $plugin->{name};
+
+    my $name;
+    if ($plugin->{name} =~ /^([\w]+)$/x) {
```

Comment:
I looked through the plugins in `stable-2.0`. There seem to be at least two plugins containing dashes. Thus maybe you should add this character to the regex?
btw.: what is the purpose of this conditional? Maybe add a comment above?